data/json/mapgen/lab/lab_surface/lab_surface_big_z4.json

```diff
@@ -0,0 +1,246 @@
+      },
+      "palettes": [ "lab_upstairs_palette" ],
+      "place_nested": [
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 24, "y": [ 45, 47 ] },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 24, "y": [ 48, 57 ], "repeat": 6 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 24, "y": [ 60, 69 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 24, "y": [ 72, 81 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 24, "y": [ 84, 93 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 30, "y": [ 36, 45 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 30, "y": [ 48, 57 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 30, "y": [ 59, 69 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 30, "y": [ 72, 81 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 89, "y": [ 45, 47 ] },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 89, "y": [ 48, 57 ], "repeat": 6 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 89, "y": [ 72, 81 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 95, "y": [ 48, 57 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 95, "y": [ 59, 69 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 95, "y": [ 72, 81 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 95, "y": [ 84, 93 ], "repeat": 7 },
+        { "chunks": [ "lab_surface_office_5x5" ], "x": 12, "y": 36 },
+        { "chunks": [ "lab_surface_2partworkspace_front_5x5" ], "x": 18, "y": 42 },
+        { "chunks": [ "lab_surface_2partworkspace_back_5x5" ], "x": 12, "y": 42 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 12, "y": 48 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 12, "y": 60 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 12, "y": 72 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 12, "y": 84 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 36 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 48 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 60 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 72 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 84 },
+        { "chunks": [ "lab_surface_2partworkspace_front_5x5" ], "x": 18, "y": 96 },
+        { "chunks": [ "lab_surface_2partworkspace_back_5x5" ], "x": 12, "y": 96 },
+        { "chunks": [ "lab_surface_2partworkspace_front_5x5" ], "x": 32, "y": 96 },
+        { "chunks": [ "lab_surface_2partworkspace_back_5x5" ], "x": 38, "y": 96 },
+        { "chunks": [ "lab_surface_office_5x5" ], "x": 103, "y": 36 },
```

Comment:
Same issue as on other floors with this nested mapgen and the door when on the east side.

![image](https://user-images.githubusercontent.com/11464/54487172-93240500-4847-11e9-8d59-36d7d210c8fb.png)